common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -91,7 +95,34 @@ def fetch_file_if_present(filename, fetch_submodules: false)
         raise Dependabot::DependencyFileNotFound, path
       end
 
+      def load_cloned_file_if_present(filename)
+        path = File.join(clone_repo_contents, filename)
+        unless File.exist?(path)
+          raise Dependabot::DependencyFileNotFound, filename
```

Comment:
```suggestion
          raise Dependabot::DependencyFileNotFound, path
```

Reply:
I think `filename` is correct here.

The user expects `requirements.txt`, not `/tmp/tmp-repo_contents_path/directory/requirements.txt`

I could dig `File.join(directory, path)`, but I'm not sure it's required? I'll add a test case (the current one is specific to `requirements.txt`)

Reply:
Looks like the existing cases of `Dependabot::DependencyFileNotFound` raise with `Pathname.new(File.join(directory, filename)).cleanpath.to_path`